master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -48,8 +48,8 @@ spec:
 | prometheusGoMetricsEnabled | Set to `false` to disable Go runtime metrics collection, which the Prometheus client does by default. This reduces the number of metrics reported, reducing Prometheus load. | boolean | boolean | `true` |
 | prometheusProcessMetricsEnabled | Set to `false` to disable process metrics collection, which the Prometheus client does by default. This reduces the number of metrics reported, reducing Prometheus load. | boolean | boolean | `true` |
 | usageReportingEnabled | Reports anonymous Calico version number and cluster size to projectcalico.org. Logs warnings returned by the usage server. For example, if a significant security vulnerability has been discovered in the version of Calico being used. | boolean | boolean | `true` |
-| failsafeInboundHostPorts | Comma-delimited list of UDP/TCP ports that Felix will allow incoming traffic to host endpoints on irrespective of the security policy.  This is useful to avoid accidently cutting off a host with incorrect configuration.  Each port should be specified as `tcp:<port-number>` or `udp:<port-number>`.  For back-compatibility, if the protocol is not specified, it defaults to "tcp".  To disable all inbound host ports, use the value `none`.  The default value allows ssh access and DHCP. | string | string | `tcp:22, udp:68` |
-| failsafeOutboundHostPorts | Comma-delimited list of UDP/TCP ports that Felix will allow outgoing traffic from host endpoints to irrespective of the security policy. This is useful to avoid accidently cutting off a host with incorrect configuration.  Each port should be specified as `tcp:<port-number>` or `udp:<port-number>`.  For back-compatibility, if the protocol is not specified, it defaults to "tcp".  To disable all outbound host ports, use the value `none`.  The default value opens etcd's standard ports to ensure that Felix does not get cut off from etcd as well as allowing DHCP and DNS. | string | string | `tcp:2379, tcp:2380, tcp:4001, tcp:7001, udp:53, udp:67` |
+| failsafeInboundHostPorts | UDP/TCP protocol/port pairs that Felix will allow incoming traffic to host endpoints on irrespective of the security policy. This is useful to avoid accidentally cutting off a host with incorrect configuration.  Each protocol/port pair should be specified as {::nomarkdown}<p><code>- protocol: [udp|tcp]<br>&nbsp;&nbsp;port: [port-number]</code></p>{:/} The default value allows SSH access, etcd, BGP and DHCP. | string | string | {::nomarkdown}<p><code> - protocol: tcp<br>&nbsp;&nbsp;port: 22<br>- protocol: tcp<br>&nbsp;&nbsp;port: 2379<br>- protocol: tcp<br>&nbsp;&nbsp;port: 2380<br>- protocol: tcp<br>&nbsp;&nbsp;port: 6666<br>- protocol: tcp<br>&nbsp;&nbsp;port: 6667<br>- protocol: udp<br>&nbsp;&nbsp;port: 68</code></p>{:/} |
```

Comment:
@2ffs2nns Thanks for adding this! Right now, we are trying to use `<variable>` to indicate a value that may change. (See style guide...). You can also see `<variable>` used here /reference/felix/configuration. 

Seems to be one extra space in front of the first `protocol`